aspnetcore/blazor/components/css-isolation.md

```diff
@@ -418,21 +405,10 @@ To opt out of how Blazor publishes and loads scoped files at runtime, use the `D
 
 ## Razor class library (RCL) support
 
-When a [Razor class library (RCL)](xref:razor-pages/ui-class) provides isolated styles, the `<link>` tag's `href` attribute points to `{STATIC WEB ASSET BASE PATH}/{PACKAGE ID}.bundle.scp.css`, where the placeholders are:
-
-* `{STATIC WEB ASSET BASE PATH}`: The static web asset base path.
-* `{PACKAGE ID}`: The class library's [package ID](/nuget/create-packages/creating-a-package-msbuild#set-properties). The package ID defaults to the project's assembly name if `<PackageId>` isn't specified in the library's project file.
-
-In the following example:
-
-* The static web asset base path is `_content/ClassLib`.
-* The class library's package ID is `ClassLib`.
+Isolated styles for components in a NuGet package or [Razor class library (RCL)](xref:razor-pages/ui-class) are automatically bundled. The bundled file:
 
-`wwwroot/index.html` (Blazor WebAssembly) or `Pages_Host.cshtml` (Blazor Server):
-
-```html
-<link href="_content/ClassLib/ClassLib.bundle.scp.css" rel="stylesheet">
-```
+* References the styles using CSS imports.
```

Comment:
Is referenced by Css import directive at the css file of referrer.

Reply:
I'll yield to Javier (and/or Dave) for the exact wording.